src/pages/office/office.html

```diff
@@ -29,28 +29,10 @@ <h4>Test Summary</h4>
     <ion-card-content class="card-content-ios" no-bounce>
       <form [formGroup]="form">
         <ion-grid class="grid">
-          <ion-row class="mes-validated-row mes-row-separator" id="route-number-card">
-            <div class="validation-bar" [class.ng-invalid]="isCtrlDirtyAndInvalid('routeNumberCtrl')"></div>
-            <ion-col class="component-label" col-32 align-self-center>
-              <label>Route number</label>
-            </ion-col>
-            <ion-col align-self-center>
-              <ion-row class="spacing-row"></ion-row>
-              <ion-row>
-                <ion-col col-32>
-                  <input #routeInput type="text" id="route" formControlName="routeNumberCtrl"
-                    class="vehicle-reg-input mes-data" numberonly
-                    [class.ng-invalid]="isCtrlDirtyAndInvalid('routeNumberCtrl')"
-                    [value]="pageState.routeNumber$ | async">
-                </ion-col>
-              </ion-row>
-              <ion-row class="validation-message-row" align-items-center>
-                <div class="validation-text" [class.ng-invalid]="isCtrlDirtyAndInvalid('routeNumberCtrl')">
-                  Enter the route number
-                </div>
-              </ion-row>
-            </ion-col>
-          </ion-row>
+
+          <route-number [routeNumber]="pageState.routeNumber$ | async" (routeNumberChange)="routeNumberChanged($event)"
+            [formGroup]="form">
+          </route-number>
 
```

Comment:
The components created for the faults were not implemented to this degree. Perhaps I should revisit this. Good work on this though. Reduction of a lot of code.